java-frontend/src/test/java/org/sonar/java/cfg/CFGTest.java

```diff
@@ -622,18 +627,21 @@ public void switch_statement_with_piledUpCases_againstDefault() {
       "void fun(int foo) { int a; switch(foo) { case 1: System.out.println(bar);case 2: System.out.println(qix);break; case 3: case 4: default: System.out.println(baz);} }");
     final CFGChecker cfgChecker = checker(
       block(
+        element(Kind.CASE_GROUP),
         element(INT_LITERAL, "1"),
         element(Tree.Kind.IDENTIFIER, "System"),
         element(Tree.Kind.MEMBER_SELECT),
         element(Tree.Kind.IDENTIFIER, "bar"),
         element(Tree.Kind.METHOD_INVOCATION)).successors(3),
       block(
+        element(Kind.CASE_GROUP),
         element(INT_LITERAL, "2"),
         element(Tree.Kind.IDENTIFIER, "System"),
         element(Tree.Kind.MEMBER_SELECT),
         element(Tree.Kind.IDENTIFIER, "qix"),
         element(Tree.Kind.METHOD_INVOCATION)).terminator(Tree.Kind.BREAK_STATEMENT).successors(0),
       block(
+        element(Kind.CASE_GROUP),
         element(INT_LITERAL, "4"),
```

Comment:
I find this a bit suspicious (i would have expected the inverted order), but it's obviously out of scope of the current ticket.